### PR TITLE
update mealPlanner footer

### PR DESCRIPTION
**Pull request by walerowicz-miam**

### CODE REVIEW

**Comment by walerowicz-miam** [comment]
* **#50** <a href="https://app.graphite.dev/github/pr/miamtech/coursesU-x-Miam-Framework/50?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* `main`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @walerowicz-miam and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: Sources/CoursesUxMiamFramework/ui/RecipeDetails/CoursesURecipeDetailsFooterView.swift [hunk 1]
```diff
@@ -51,8 +55,47 @@ public struct CoursesURecipeDetailsFooterView: RecipeDetailsFooterProtocol {
...
+        var priceByPerson: Double
+        var totalPrice : Double
+        
+        public var body: some View {
+            HStack(spacing: 0) {
+                Text(priceByPerson.currencyFormatted)
+                .foregroundColor(Color.black)
+                .coursesUFontStyle(style: CoursesUFontStyleProvider.sharedInstance.bodyBigBoldStyle)
+                
+                Text(" par personne")
```
Comment (diarmuidmcg) [inline Sources/CoursesUxMiamFramework/ui/RecipeDetails/CoursesURecipeDetailsFooterView.swift:69]: c'est Localization.price.perGuest.localised

File: Sources/CoursesUxMiamFramework/ui/RecipeDetails/CoursesURecipeDetailsFooterView.swift [hunk 2]
```diff
@@ -51,8 +55,47 @@ public struct CoursesURecipeDetailsFooterView: RecipeDetailsFooterProtocol {
...
+                    .coursesUFontStyle(style: CoursesUFontStyleProvider.sharedInstance.bodySmallStyle)
+                Spacer()
+                HStack(spacing: 0) {
+                    Text("Soit ")
+                        .foregroundColor(Color.black)
+                        .coursesUFontStyle(style: CoursesUFontStyleProvider.sharedInstance.bodyStyle)
+                    YellowSubtext(text: totalPrice.currencyFormatted, fontStyle: CoursesUFontStyleProvider.sharedInstance.bodyBigBoldStyle, imageWidth: 55)
+                    Text("le repas")
+                        .foregroundColor(Color.black)
+                        .coursesUFontStyle(style: CoursesUFontStyleProvider.sharedInstance.bodyStyle)
```
Comment (diarmuidmcg) [inline Sources/CoursesUxMiamFramework/ui/RecipeDetails/CoursesURecipeDetailsFooterView.swift:80]: c'est possible utilise Localization.price.mealCost(cost: ).localised ?

**Review by diarmuidmcg — COMMENTED** [review]
localization

File: Sources/CoursesUxMiamFramework/ui/RecipeDetails/CoursesURecipeDetailsFooterView.swift (hunk 2 above)
Comment (walerowicz-miam) [reply Sources/CoursesUxMiamFramework/ui/RecipeDetails/CoursesURecipeDetailsFooterView.swift:80]: le design est différent entre les texts

**Review by diarmuidmcg — APPROVED** [review]

**Pull request merged**